### PR TITLE
Code smells

### PR DESCRIPTION
**Pull request by avikalpg**
## Summary by CodeRabbit

- **New Features**
	- Updated the Banner display mechanism for enhanced user experience.
	- Improved user logout functionality with enhanced accessibility.
	- Enhanced user profile update functionality to support appending aliases.
	- Restructured user repository retrieval logic for GitHub and Bitbucket for improved efficiency.
- **Bug Fixes**
	- Fixed a scope issue in the data processing function to ensure accurate mode calculations.
- **Refactor**
	- Adjusted the logic for updating aliases in user profile handling to preserve old aliases.
- **Style**
	- Enhanced Navbar accessibility by adding semantic roles to elements.

### CODE REVIEW

**Comment by avikalpg** [comment]
Do not merge this yet.

File: views/Navbar.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ const Navbar = (props: { transparent: boolean }) => {
 				</li>
 			</ul>
 			{/* Mobile Button */}
-			<div onClick={changeNavbar}
+			<div onClick={changeNavbar} role='button'
 				className={
 					'block sm:hidden z-10' + (scrollDown || !props.transparent ? ' text-secondary-dark' : ' text-primary-light')
 				}
```
Comment (avikalpg) [inline views/Navbar.tsx:115]: Can I use an <input> element here instead of the div?

Comment (tapishr) [reply views/Navbar.tsx:115]: Who you asking? What difference will an element make?

**Comment by avikalpg** [comment]
@tapishr Please review this now and merge it if it looks good.

**Review by MuskanPaliwal — APPROVED** [review]

**Pull request merged**